src/components/error-boundary/error-boundary.tsx

```diff
@@ -13,7 +27,7 @@ export default class ErrorBoundary extends React.Component {
         };
     }
 
-    componentDidCatch(error, errorInfo) {
+    componentDidCatch(error: object | null, errorInfo: ErrorBoundaryErrorInfo) {
```

Comment:
We can point to a part of interface, like `ErrorBoundaryState['error']`

Reply:
or we can define the type out of the interface and point it to the defined type for error `ErrorType`

Reply:
makes sense, updated in this PR